internals/secrethub/org_inspect_test.go

```diff
@@ -107,29 +107,42 @@ func TestOrgInspectCommand_Run(t *testing.T) {
 		},
 		"get org error": {
 			orgService: fakeclient.OrgService{
-				Getter: fakeclient.OrgGetter{
-					Err: testErr,
+				GetFunc: func(name string) (*api.Org, error) {
+					return nil, testErr
 				},
 			},
 			err: testErr,
 		},
 		"list org members error": {
 			orgService: fakeclient.OrgService{
-				MemberService: &fakeclient.OrgMemberService{
-					Lister: fakeclient.OrgMemberLister{
-						Err: testErr,
+				MembersService: &fakeclient.OrgMemberService{
+					ListFunc: func(org string) ([]*api.OrgMember, error) {
+						return nil, testErr
 					},
 				},
+				GetFunc: func(name string) (*api.Org, error) {
+					return &api.Org{}, nil
+				},
 			},
 			err: testErr,
 		},
 		"list repos error": {
 			orgService: fakeclient.OrgService{
-				MemberService: &fakeclient.OrgMemberService{},
+				MembersService: &fakeclient.OrgMemberService{
+					ListFunc: func(org string) ([]*api.OrgMember, error) {
+						return nil, nil
+					},
+				},
+				GetFunc: func(name string) (*api.Org, error) {
```

Comment:
here

Reply:
This one is required.

Reply:
Can you explain why? It wasn't supplied before, right?

Reply:
Sorry, I should have clarified all this when opening the PR, so here is my explanation:
Before the change, we were using structs for mocks, for example the `OrgGetter`: 

```go
type OrgGetter struct {
	ArgName    string
	ReturnsOrg *api.Org
	Err        error
}


// Get saves the arguments it was called with and returns the mocked response.
func (g *OrgGetter) Get(name string) (*api.Org, error) {
	g.ArgName = name
	return g.ReturnsOrg, g.Err
}
```
If the the OrgGetter was not initialized it returned the zero values `(nil, nil)`.

In contrast, when using functions, an uninitialized `func` field has the value of `nil` and panics when `OrgService.Get` is called.
```go
func (s *OrgService) Get(name string) (*api.Org, error) {
	return s.GetFunc(name)
}
```

I thought about the drawback that now we have to initialize more fields when testing, however I realized it might be actually be an advantage as it forces the tester to initialize all used client functions explicitly and makes it more clear what the test is doing (in my opinion).

Reply:
Yes that's great. It's totally fine for tests to fail/panic if a developer hasn't set all the required fields of the fake. IMO, that's actually a feature, not a bug.

Reply:
> I thought about the drawback that now we have to initialize more fields when testing, however I realized it might be actually be an advantage as it forces the tester to initialize all used client functions explicitly and makes it more clear what the test is doing (in my opinion).

I agree that it's an advantage that we're now explicit about that a function is called and what is returned :+1: